framework/tlvf/AutoGenerated/src/tlvf/test/tlvVarList.cpp

```diff
@@ -285,12 +285,12 @@ bool tlvTestVarList::init()
     m_type = (uint8_t*)m_buff_ptr__;
     if (!m_parse__) *m_type = 0x1;
     m_buff_ptr__ += sizeof(uint8_t) * 1;
-    m_var0 = (uint16_t*)m_buff_ptr__;
-    m_buff_ptr__ += sizeof(uint16_t) * 1;
-    if(m_length && !m_parse__){ (*m_length) += sizeof(uint16_t); }
     m_length = (uint16_t*)m_buff_ptr__;
     if (!m_parse__) *m_length = 0;
     m_buff_ptr__ += sizeof(uint16_t) * 1;
+    m_var0 = (uint16_t*)m_buff_ptr__;
+    m_buff_ptr__ += sizeof(uint16_t) * 1;
+    if(m_length && !m_parse__){ (*m_length) += sizeof(uint16_t); }
     m_simple_list_length = (uint16_t*)m_buff_ptr__;
```

Comment:
I think this commit can be squashed to the previous one (nitpick)
In general I think that splitting to autogenerated commits is worth it only if there are a lot of autogenerated changes.

Reply:
Note that github doesn't keep track of which commit was open when you type a review comment. So "this commit" is fairly meaningless.

I agree that splitting off autogenerated commits is not very valuable. Conversely, I found it very useful when reviewing #139 to have the tlvf.py change together with the source code changes, because you immediately see the effect.